CMakeLists.txt

```diff
@@ -398,6 +404,7 @@ target_link_libraries(vcpkglib
         cmakerc::locales
         Threads::Threads
     PRIVATE
+        ${LIBANDROID_SPAWN}
         ${CPP_ATOMIC_LIBRARY}
 )
 
```

Comment:
Instead of doing variables like this, I think this should be:

```
if(DEFINED ENV{TERMUX_VERSION})
  # pkg install libandroid-spawn
  target_link_libraries(vcpkglib PRIVATE android-spawn)
endif()
```

Reply:
Yes thank you, also I've already updated it to allow non-Termux builds.